### ISSUE TITLE
terminal.ansiBrightBlue is of the same color as the background

### ISSUE THREAD

**Issue by LiquidPL** [kleber-swf/vscode-ocean-dark-extended-theme#3]
This leads to some weird stuff like here:

![image](https://user-images.githubusercontent.com/991831/36114330-0b16fa28-1030-11e8-8fe6-c601a1cf288a.png)

as opposed to default theme:

![image](https://user-images.githubusercontent.com/991831/36114350-1c2ffb3e-1030-11e8-83aa-c04bb4576a8d.png)

**Comment by marpala** [comment]
Same problem here

**Comment by dotiful** [comment]
same. as a temporary workaround, i fixed it like this
```json
"workbench.colorCustomizations":
  "terminal.ansiBrightBlue": "#8fa1b3"
}
```

**Comment by kleber-swf** [comment]
Sorry for the delay, guys.
Just published the version 1.5.0 fixing this issue.
Actually all terminal colors were revised.
Thanks for the patience.
And special thanks to @jonathansimmons for the PR and @dotiful for the quick fix!
